src/utils/DynamicContent/generators/ProgressBarContent.ts

```diff
@@ -57,4 +60,8 @@ export class ProgressBarContent implements DynamicProgressBarContent {
 	public get isLateProgress(): boolean {
 		return this._isLateProgress;
 	}
+
+	public get dramaTextIsVisible(): boolean {
+		return this._dramaTextIsVisible;
+	}
```

Comment:
> As far as I can see the progress bars themselves don't do anything yet with the drama text (flag or text), so that's also something that should be implemented in this PR.

as far as I understood this is done in banner tickets and not in this PR because there is no design definition or anything like that in the ticket